### PR TITLE
[HomeTask2] | added javascript snippets for HomeTask2

### PR DESCRIPTION
**Pull request by bommas12**
### @SaiTeja2904 Please review the snippets added for HomeTask2

### CODE REVIEW

File: HomeTask2/basicOp.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * @param {string} operation Any character from , '+','-','*','/'
+ * @param {number} value1 number 1
+ * @param {number} value2 number 2
+ * @example basicOp('+', 4, 7)
+ * 
+ * @returns {number} evaluation of the expression
+ */
+
+function basicOp(operation, value1, value2) {
+    return eval(value1 + operation + value2)
```
Comment (SaiTeja2904) [inline HomeTask2/basicOp.js]: It is not recommended to use eval function as it has security issues. Hence develop another approach that would perform the same.

File: HomeTask2/rentalCarCost.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    const DISCOUNT_FOR_7DAYS = 50;
+    const DISCOUNT_FOR_3DAYS = 20;
+    let totalCost = numberOfDays * COST_PER_DAY;
+    if (numberOfDays >= 7) {
+        return totalCost - DISCOUNT_FOR_7DAYS;
+    }
+    if (numberOfDays >= 3) {
+        return totalCost - DISCOUNT_FOR_3DAYS;
+    }
+    return totalCost;
```
Comment (SaiTeja2904) [inline HomeTask2/rentalCarCost.js:19]: Do not use multiple return statements in a single function. Save it under one variable and return it at the end of the function.

**Review by SaiTeja2904 — CHANGES_REQUESTED** [review]

File: HomeTask2/rentalCarCost.js (hunk 2 above)
Comment (bommas12) [reply HomeTask2/rentalCarCost.js:19]: I made the changes as requested.
1. removed eval from basicOp
2.  removed multiple returns from rentalCarCost

File: HomeTask2/basicOp.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * @param {number} value2 number 2
+ * @example basicOp('+', 4, 7)
+ * 
+ * @returns {number} evaluation of the expression
+ */
+
+function basicOp(operation, value1, value2) {
+    const operators = {
+        '+': (number1, number2) => {
+            return number1 + number2;
```
Comment (SaiTeja2904) [inline HomeTask2/basicOp.js]: If you just do a simple transformation or calculation, you need not put a return statement. Instead, you can do it as (num1, num2) => num1 + num2. Can avoid usage of unnecessary brackets and return statements.

File: HomeTask2/basicOp.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        },
+        '-': (number1, number2) => {
+            return number1 - number2;
+        },
+        '/': (number1, number2) => {
+            return number1 / number2;
```
Comment (SaiTeja2904) [inline HomeTask2/basicOp.js:20]: Implement an addition check If number2 is zero.

**Review by SaiTeja2904 — CHANGES_REQUESTED** [review]

File: HomeTask2/basicOp.js (hunk 3 above)
Comment (bommas12) [reply HomeTask2/basicOp.js]: Done changes as per the request

File: HomeTask2/basicOp.js (hunk 4 above)
Comment (bommas12) [reply HomeTask2/basicOp.js:20]: check for number2 is 0 is added

**Review by SaiTeja2904 — APPROVED** [review]

**Pull request merged**